react-list.es6

```diff
@@ -224,14 +224,31 @@ module.exports = class ReactList extends Component {
     this.frameRequested = window.requestAnimationFrame(this.updateFrame);
   }
 
+  waitForFrameUpdate() {
```

Comment:
@ibash `onFrameUpdate` gets called more times than `onScroll`. Specifically
1) it also gets when you hover over a `ThreadListItem`.
2) It also gets called when more items are added to the list.

Also:
1) `updateFirstItemRank` is a synchronous costless operation so it doesn't matter if we are firing onScroll often.
2) We are not seeing all the callback to waitForFrameUpdate waiting for the same frame to be updated because onScroll events are happening at the bigger interval. So adding debouncer wouldn't help.